scss/inc/_forms.scss

```diff
@@ -342,7 +342,7 @@ input[type="password"].viewable-hiddenbox-input {
         }
     }
 
-    + .form-error {
+    & + .form-error {
```

Comment:
This does nothing. The compiled CSS will be identical for the sibling selector.

Reply:
I pushed the final solution with less nested rules and still support old code that it works